utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/ConfluentSchemaRegistryClient.scala

```diff
@@ -3,11 +3,14 @@ package pl.touk.nussknacker.engine.avro.schemaregistry.confluent.client
 import cats.data.Validated
 import cats.data.Validated.{invalid, valid}
 import com.typesafe.scalalogging.LazyLogging
+import io.confluent.kafka.schemaregistry.ParsedSchema
+import io.confluent.kafka.schemaregistry.avro.AvroSchema
 import io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException
 import io.confluent.kafka.schemaregistry.client.{SchemaRegistryClient => CSchemaRegistryClient}
-import pl.touk.nussknacker.engine.avro.AvroUtils
 import pl.touk.nussknacker.engine.avro.schemaregistry._
 import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.ConfluentUtils
+import pl.touk.nussknacker.engine.kafka.SchemaRegistryClientKafkaConfig
+
 import scala.collection.JavaConverters._
 
 trait ConfluentSchemaRegistryClient extends SchemaRegistryClient with LazyLogging {
```

Comment:
can we make method
```
def client: CSchemaRegistryClient
```
protected?
or enforce in other way, that noone directly calls original client? (if not, maybe we should implement delegate to `io.confluent.kafka.schemaregistry.client.SchemaRegistryClient` and map to new schema type there)?

Reply:
Let's do it in separate PR